wezterm/src/main.rs

```diff
@@ -29,6 +29,10 @@ struct Opt {
     #[structopt(short = "n")]
     skip_config: bool,
 
+    /// Specify the configuration file to use, overrides WEZTERM_CONFIG_FILE
+    #[structopt(long = "config-file", parse(from_os_str))]
```

Comment:
you can add something like `conflicts_with = "skip_config"` in here to have clap/structopt error out if both `-n` and `--config-file` are used together.

I mention this because I have been bitten by `WEZTERM_CONFIG_FILE=/tmp/issueXXX.lua wezterm -n` not doing what I expect a few times while investigating issues!

Reply:
I tried to add `conflicts_with` on `config_file` structopt, but when testing it doesn't seem to work at all, I can do `./target/debug/wezterm -n --config-file conf.lua` without errors :confused:

Reply:
I wonder if it wants `skip-config` instead of `skip_config`; `structopt` presumably translates the name to use hyphens to pass down to the underlying `clap` crate.

Reply:
that was it!